textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/common/KeywordPredicate.java

```diff
@@ -50,7 +52,6 @@ public KeywordPredicate(String query, List<Attribute> attributeList, Analyzer an
         }
     }
 
-    @Override
     public boolean satisfy(ITuple tuple) {
 
         //This method is necessary for the interface implementation, and it's really not used.
```

Comment:
The original `satisfy()` was added here to meet the interface requirement.  Since the method has been removed from the interface, should we also remove this method here?  @prakul ?